prototypes/user-preferences/src/App.css

```diff
@@ -0,0 +1,88 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: row;
+  padding-top: 30px;
+  align-items: start;
+  justify-content: space-around;
+  font-size: calc(10px + 2vmin);
+  background-color: #282c34;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@media (min-width: 768px) {
+  .App-header {
+    background-image: linear-gradient(to top, #dfa94b, #bb545e);
+  }
+}
+
+@media (min-width: 5px) {
+  .m-0 {
+    /*box-shadow: 10px 10px 5px pink;*/
+    box-shadow: 10px 10px 1px black;
+    background-color: peru;
+    height: 60px;
+    align-content: center;
+  }
+}
+
+.Panel {
+  background-color: pink;
+}
+
+.p-panel-title {
+  font-size: 1.5rem;
+  /* text-align: center; */
```

Comment:
Remove this

Reply:
remove this because it's rendered redundant from another CSS <p> style?